Update library_html5.js

Jonathhhan opened the pull request:
https://github.com/emscripten-core/emscripten/issues/15517

juj commented:
Sorry, we should not do a change like this. That would break a lot of existing users.

The intent of these functions is to provide a pass-through import of the HTML5 DOM Events over to Wasm. If you need to do adjustments of any sorts to the coordinates, they should be done in the wasm code side, instead of in the HTML5 layer (unless they are there to clearly work around a specific browser bug?)

Another issue is that the code is referencing `target.width` and `target.height`. It seems to be assuming that `target` is a HTMLCanvasElement, but these HTML5 event handlers are general and can be called to set up receiving events for any HTML5 DOM element that will not have `.width` or `.height` members. As a result, the `targetX` and `targetY` fields would compute `undefined` for any non-canvas targets.

It looks like the issue that is being tried to solve here is to convert the browser CSS coordinates to canvas local pixel coordinates? Such conversion should be up to the application developer to perform in Wasm land - that way these event handler functions will remain generic and fit for any user/any purpose. Check out the Emscripten functions `emscripten_get_canvas_element_size()` and `emscripten_get_element_css_size()` on how to get the size data into Wasm land.

Jonathhhan commented:
@juj thank you for the hints and the clarification. for now (and myself) i replaced target.width with Module['canvas'].width. and in browser_library.js i replaced emscripten_get_canvas_size() with emscripten_get_canvas_element_size()...

> It looks like the issue that is being tried to solve here is to convert the browser CSS coordinates to canvas local pixel coordinates? 

Yes, I will have a look how to do that in Open Frameworks.
